docs/content/concepts/layer/control/catalog/_index.md

```diff
@@ -85,157 +88,271 @@ See [examples](#content-examples) of OSCAL catalogs.
+
+OSCAL parameters are worth special coverage as a topic in themselves. As shown here, they can and commonly do have internal structure and properties of their own, sometimes specific to a particular application, requirement or use case. For example, in this data set, `<prop>` elements are given to provide extra information -- `alt-identifier` values -- helping assure parameters' correspondences across versions of SP 800-53. Consumers of OSCAL content can determine where and how such information is useful to them, and ignore or discard it when it constitutes noise.
+
+#### Properties
+
+After all parameters are defined, a control may be provided with a set of `<prop>` elements, each one of which specifies a single *property*, representing a name-value pair, in OSCAL. The particular meaning of each property is subject to definition by its originator (who defines the name and namespace and attributes the semantics), as interpreted by recipients in their own operational (semantic) context -- and is therefore subject to *negotiation*, *contracting*, and ultimately to *standardization* (external specification by third parties). Industry-standard names and value spaces for common properties -- along with technological platforms for validating them -- are being defined to help OSCAL developers and consumers leverage their common work, while also rapidly exploring and describing the problem space.
+
+{{<highlight xml "nowrap=false" >}}
+<prop name="label" value="AC-1"/>
+<prop name="label" class="sp800-53a" value="AC-01"/>
+<prop name="sort-id" value="ac-01"/>
+{{< / highlight >}}
+
+In this representation of AC-1, three properties are given by the data publisher. Each of these properties may be fairly considered to be a "convenience value": for example, two properties provides data consumers with reliable values for a control label "as published" in different renditions of the catalog. An extra identifier is also provided for sorting. As shown here, `<prop>` elements can also be qualified by `class` to offer further distinctions from neighbor properties.
+
+In OSCAL, the values of properties must be atomic values, such as strings, numbers or dates, without Markdown (for example) or other kinds of formatting. To extend properties to provide for values with more complex object types represents a challenge to be met, because any representation of data at this level must be in a form as widely accessible as possible across the greatest range of consumers (languages, platforms, serialization formats, and information modeling and programming technologies and paradigms). To mitigate this limitation, where simple qualification by assigning name/value pairs is not enough, controls (and other structures) in OSCAL can also be extended using other generic elements such as `<link>` and `<part>`.
```

Comment:
Isn't it technically within OSCAL `markup-multiline` datatype and just happens to overlap with Markdown when present in YAML, but still not considered Markdown? I understand this is a little in the weeds for a concepts page, but this can muddy the waters and lead to confusion later.